### PR TITLE
For #2567 - Adjusts size / position of autocomplete button 🔄

### PR DESCRIPTION
**Pull request by boek**
image

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteQuickAddPopup.kt [hunk 1]
```diff
@@ -42,11 +40,11 @@ class AutocompleteQuickAddPopup(context: Context, url: String) : PopupWindow() {
...
 
     fun show(anchor: View) {
-        val y = (anchor.height * Y_POS_MULTIPLIER).roundToInt()
-        super.showAtLocation(anchor, Gravity.CENTER_HORIZONTAL or Gravity.TOP, 0, y)
+        super.showAsDropDown(anchor, X_POS_OFFSET, Y_POS_OFFSET)
     }
 
     companion object {
-        const val Y_POS_MULTIPLIER = 1.5
+        const val X_POS_OFFSET = -10
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteQuickAddPopup.kt]: \[detekt\] INFO: \[style/MagicNumber\] Report magic numbers\. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is\. It's better to declare such numbers as constants and give them a proper name\. By default, \-1, 0, 1, and 2 are not considered to be magic numbers\.

Comment (boek) [reply app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteQuickAddPopup.kt]: But sputnik, these are constants?

Comment (boek) [reply app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteQuickAddPopup.kt]: maybe they need to be private?

File: app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteQuickAddPopup.kt [hunk 2]
```diff
@@ -42,11 +40,11 @@ class AutocompleteQuickAddPopup(context: Context, url: String) : PopupWindow() {
...
 
     fun show(anchor: View) {
-        val y = (anchor.height * Y_POS_MULTIPLIER).roundToInt()
-        super.showAtLocation(anchor, Gravity.CENTER_HORIZONTAL or Gravity.TOP, 0, y)
+        super.showAsDropDown(anchor, X_POS_OFFSET, Y_POS_OFFSET)
     }
 
     companion object {
-        const val Y_POS_MULTIPLIER = 1.5
+        private const val X_POS_OFFSET = -10
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteQuickAddPopup.kt]: \[detekt\] INFO: \[style/MagicNumber\] Report magic numbers\. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is\. It's better to declare such numbers as constants and give them a proper name\. By default, \-1, 0, 1, and 2 are not considered to be magic numbers\.

**Pull request merged**